crowdin.yml

```diff
@@ -1,3 +1,3 @@
 files:
-  - source: /src/locales/default.json
+  - source: /src/locales/en-US.json
```

Comment:
Is it possible to not change this?

Reply:
I would have asked you how much of an impact that actually has. I can revert this, sure. :D